spring-cloud-dataflow-platform-cloudfoundry/pom.xml

```diff
@@ -33,25 +33,13 @@
 		  <optional>true</optional>
 		</dependency>
 		<dependency>
-			<groupId>io.pivotal.spring.cloud</groupId>
-			<artifactId>spring-cloud-services-starter-config-client</artifactId>
-			<version>${spring-cloud-services.version}</version>
-			<scope>runtime</scope>
-			<exclusions>
-				<exclusion>
-					<groupId>org.springframework.cloud</groupId>
-					<artifactId>spring-cloud-cloudfoundry-connector</artifactId>
-				</exclusion>
-				<exclusion>
-					<groupId>org.springframework.cloud</groupId>
-					<artifactId>spring-cloud-spring-service-connector</artifactId>
-				</exclusion>
-			</exclusions>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-config</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>io.pivotal.spring.cloud</groupId>
-			<artifactId>spring-cloud-sso-connector</artifactId>
-			<version>${spring-cloud-sso.version}</version>
+			<groupId>org.springframework.cfenv</groupId>
+			<artifactId>spring-cfenv-boot</artifactId>
+			<version>1.0.0.BUILD-SNAPSHOT</version>
```

Comment:
Version to dep management.

Reply:
You know, I meant putting `spring-cfenv-boot` in a dependency management section in a parent pom so that in a module we can just add dependency without specifying version.

Reply:
Right, there is other deps defined like this. Maybe we'll polish those poms with a different commit.